src/python/pants/backend/python/goals/lockfile.py

```diff
@@ -176,6 +176,18 @@ async def generate_lockfile(
                     "--style=universal",
                     "--resolver-version",
                     "pip-2020-resolver",
+                    # PEX files currently only run on Linux and Mac machines; so we hard code this
```

Comment:
This is the one code change in all this noise version bump / `./build-support/bin/generate_all_lockfiles.sh --all` noise.